lib/feature/network/bottom_sheets/switch_to_this_network_sheet.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:app/app/service/service.dart';
+import 'package:app/di/di.dart';
+import 'package:app/feature/network/network.dart';
+import 'package:app/generated/generated.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:ui_components_lib/ui_components_lib.dart';
+
+/// Helper function to show [SwitchToThisNetworkSheet].
+ModalRoute<void> showSwitchToThisNetworkSheet({
+  required String connectionId,
+}) {
+  return commonBottomSheetRoute(
```

Comment:
It's better to use `showCommonBottomSheet` method to display sheet, because method, returning `ModalRoute` better to use when you want to do `Navigator..pop()..push(sheetRoute)`

Reply:
🤟